html/global_attributes.json

```
@@ -1108,7 +1108,14 @@
               }
             ],
             "firefox_android": {
-              "version_added": false
+              "version_added": "68",
```

Based on the feature in the desktop Firefox, I'm guessing this has been in the Android version for some time. Let's set this to true:

```suggestion
              "version_added": true,
```

---

Why? I haven't seen any indication that this was enabled before: https://bugzilla.mozilla.org/show_bug.cgi?id=1631681#c0

---

Ah, thank you @evilpie. I misunderstood the original PR description ("as of Version 68") to mean that it was tested in the current version and no others. I'll merge it as is.